subprojects/messaging/src/test/groovy/org/gradle/internal/serialize/DefaultSerializerRegistrySpec.groovy

```diff
@@ -0,0 +1,279 @@
+
+    static class Serializable55 extends SerializableParent {}
+
+    static class Serializable56 extends SerializableParent {}
+
+    static class Serializable57 extends SerializableParent {}
+
+    static class Serializable58 extends SerializableParent {}
+
+    static class Serializable59 extends SerializableParent {}
+
+    static class Serializable60 extends SerializableParent {}
+
+    static class Serializable61 extends SerializableParent {}
+
+    static class Serializable62 extends SerializableParent {}
+
+    static class Serializable63 extends SerializableParent {}
+
+    static class Serializable64 extends SerializableParent {}
+
+    static class Serializable65 extends SerializableParent {}
+
+    static class Serializable66 extends SerializableParent {}
+
+    static class Serializable67 extends SerializableParent {}
+
+    static class Serializable68 extends SerializableParent {}
+
+    static class Serializable69 extends SerializableParent {}
+
+    static class Serializable70 extends SerializableParent {}
+
+    static class Serializable71 extends SerializableParent {}
+
+    static class Serializable72 extends SerializableParent {}
+
+    static class Serializable73 extends SerializableParent {}
+
+    static class Serializable74 extends SerializableParent {}
+
+    static class Serializable75 extends SerializableParent {}
+
+    static class Serializable76 extends SerializableParent {}
+
+    static class Serializable77 extends SerializableParent {}
+
+    static class Serializable78 extends SerializableParent {}
+
+    static class Serializable79 extends SerializableParent {}
+
+    static class Serializable80 extends SerializableParent {}
+
+    static class Serializable81 extends SerializableParent {}
+
+    static class Serializable82 extends SerializableParent {}
+
+    static class Serializable83 extends SerializableParent {}
+
+    static class Serializable84 extends SerializableParent {}
+
+    static class Serializable85 extends SerializableParent {}
+
+    static class Serializable86 extends SerializableParent {}
+
+    static class Serializable87 extends SerializableParent {}
+
+    static class Serializable88 extends SerializableParent {}
+
+    static class Serializable89 extends SerializableParent {}
+
+    static class Serializable90 extends SerializableParent {}
+
+    static class Serializable91 extends SerializableParent {}
+
+    static class Serializable92 extends SerializableParent {}
+
+    static class Serializable93 extends SerializableParent {}
+
+    static class Serializable94 extends SerializableParent {}
+
+    static class Serializable95 extends SerializableParent {}
+
+    static class Serializable96 extends SerializableParent {}
+
+    static class Serializable97 extends SerializableParent {}
+
+    static class Serializable98 extends SerializableParent {}
+
+    static class Serializable99 extends SerializableParent {}
+
+    static class Serializable100 extends SerializableParent {}
```

Comment:
GitHub Copilot coming in clutch here for making writing all of this significantly less painful.